src/validation.cpp

```diff
@@ -5037,6 +5038,14 @@ int VersionBitsTipStateSinceHeight(const Consensus::Params& params, Consensus::D
 
 static const uint64_t MEMPOOL_DUMP_VERSION = 1;
 
+// Check if an amount is a valid fee delta.
+//
+// Like MoneyRange(...), but allows for negative money amounts.
+static bool FeeDeltaRange(const CAmount fee_delta)
```

Comment:
This helper could be moved to the amount module?

Reply:
Good idea! Done! :)